docs/index.rst

```diff
@@ -201,6 +201,14 @@ Sending messages in bulk
     # Or (send_message parameters include: messages, dry_run, app)
     FCMDevice.objects.send_message(Message(...))
 
+    Sending messages raises all the errors that ``firebase-admin`` raises, so make sure
```

Comment:
That part of docs was missed from `index.rst` So I just synchronize content of `docs/index.rst` and `README.rst`